### PR TITLE
prep doc work for beta release

### PR DESCRIPTION
**Pull request by tardyp**
Draft email for annoucement:

Don't hesitate to fix my prose.

```
I am very happy too announce the release of buildbot 0.9.0 beta 1.

This is a great milestone to the Buildbot Nine project, the result of a lot of work by a team of developers from all the world, during the last 3 years.

As the diff stats says, the change compared to eight is huge:
1160 files changed, 88098 insertions(+), 58070 deletions(-)

So we must go through a beta phase before releasing a definitive 0.9.0 version.
Buildbot nine already has several installation in production, and lots of positive feedback are coming to us.

Please join us, and report any bugs or issues in http://trac.buildbot.net
 
Please find the release notes for all technical details of what's new.

[...]

The UI demo server is available here:
http://nine.buildbot.net/
      
I would like to thank all the contributors that made this release possible, this is sorted by number of commit, but every line of code means a lot for us:
          
git log v0.8.10..HEAD --no-merges   --format='%aN' | sort  |uniq -c |sort -nr
    652 Dustin J. Mitchell
    457 Pierre Tardy
    303 Mikhail Sobolev
    137 Benoît Allard
     93 Chase Zhang
     53 Jared Grubb
     50 Xavier Delannoy
     33 Jonas Pommerening
     30 Andras Toth
     28 Sean Kelly
     28 Michael Mayorov
     27 Edmund Wong
     24 chase
     23 delanne
     22 Ion Alberdi
     19 Vladimir Rutsky
     18 Elmir Jagudin
     18 Buck Golemon
     16 Vitali Lovich
     16 Oliver Palmer
     14 Tom Prince
     14 MaximeSangoi
     11 Dylan Rajaratnam
     10 Sebastien Fusilier
     10 Adi Roiban
      9 Jørn Lomax
      9 Esben Haabendal
      7 Sebastien Savrimoutou
      7 Dave B.
      6 Tsubashi
      5 Роман Донченко
      5 Vadim Zeitlin
      5 tothandras
      5 Olof Johansson
      5 Markus Wanner
      5 Kassem Sandarusi
      4 Stefan Gangefors
      4 rdb
      4 Prasoon Shukla
      4 Marek Sezemsky
      4 Joshua Clayton
      4 Jeppe Pihl
      4 Hashem Nasarat
      4 Andriy Senkovych
      3 Thomas Wood
      3 Olivier Monnier
      3 Michal Sojka
      3 Kevin Funk
      3 Kálmán Viktor
      3 Daniel Collin
      3 András Tóth
      2 VZ
      2 Vadim Rutkovsky
      2 Steve
      2 stanzgy
      2 Rahul
      2 Quentin Pradet
      2 Peter Lemenkov
      2 Olivier Favre
      2 Kevin Chang
      2 InitHello
      2 Greg MacDonald
      2 Florian Bruhin
      2 Dan Kegel
      2 Cyril Jouve
      2 Cyril
      2 Cody P Schafer
      2 Anish Bhatt
      2 Andrew Hammond
      1 Varun Sharma
      1 Thomas Vander Stichele
      1 stevepeak
      1 socantre
      1 Sezi
      1 Rafał
      1 Pier Luigi Fiorini
      1 Peter Dave Hello
      1 Olly Betts
      1 Nikolai Kristiansen
      1 Naveen
      1 maximesangoi
      1 Matvey Kruglov
      1 Mark Rowe
      1 Luke Bakken
      1 koobs
      1 heinrich5991
      1 dvinella
      1 Dave B
      1 Chris Rebert
      1 Bo Shi
      1 Anders Bennehag

```

### CODE REVIEW

File: master/docs/relnotes/0.9.0b1.rst [hunk 1]
```diff
@@ -444,7 +455,7 @@ git log itself:
 
 .. code-block:: bash
 
-   git log v0.8.10..master
+   git log v0.8.10..v0.9.0b1
```
Comment (tardyp) [inline master/docs/relnotes/0.9.0b1.rst]: good luck with that though :)

**Comment by sa2ajj** [comment]
I'd exclude merge commits :)

**Comment by sa2ajj** [comment]
Yeah, that's more like it.

:+1:

**Comment by tardyp** [comment]
Lets wait one more day, and declare beta tomorrow.

**Comment by djmitche** [comment]
+1 ship it!

**Pull request merged**